src/components/Chart/LineChart.jsx

```diff
@@ -3,7 +3,11 @@ import PropTypes from 'prop-types'
 import * as d3 from 'utils/d3'
 import { max, minBy, keyBy, memoize } from 'lodash'
 import styles from 'components/Chart/LineChart.styl'
-import Tooltip from 'components/Chart/Tooltip'
+
+const getCSSPropertyValue = memoize(varName => {
```

Comment:
We already have a [`getCssVariableValue`](https://github.com/cozy/cozy-ui/blob/master/react/utils/color.js#L1) function in cozy-ui. But it's not memoized, it can be a good improvement.